handbook/operations.md

```diff
@@ -0,0 +1,63 @@
+# Operations
+
+## People Ops
+
+### Hiring a new team member
+
+1. Reach out to Mike and Zach and give them an opportunity to meet the candidate.
+
+2. Research compensation using [Pave](https://www.pave.com), making sure to adjust for geographical factors. *If you're unsure of your findings, ask for help from Mike.*
```

Comment:
What are "geographical factors"? I think it would be helpful to include some examples.

Reply:
replace "geographical factors" with "cost of living where the candidate will do the work."